lib/views/circle_view.dart

```diff
@@ -10,13 +10,25 @@ class CircleView extends StatelessWidget {
 
   final Border border;
 
-  CircleView({this.size, this.color, this.boxShadow, this.border});
+  final Image buttonImage;
+
+  final String buttonText;
+
+  CircleView(
+      {this.size,
+        this.color = Colors.transparent,
+        this.boxShadow,
+        this.border,
+        this.buttonImage,
+        this.buttonText = ""});
 
   @override
   Widget build(BuildContext context) {
     return Container(
       width: size,
       height: size,
+      child:
+      Center(child: (buttonImage != null) ? buttonImage : Text(buttonText)),
```

Comment:
Maybe we could set that Text(buttonText) "default value" for buttonImage.
This way here we would have just
> child: Center(child: buttonImage),